src/test/java/spoon/test/annotation/AnnotationTest.java

```diff
@@ -296,7 +297,7 @@ public void testAnnotatedElementTypes() throws Exception {
 		assertTrue(annotations.get(0).getAnnotatedElement().equals(enumeration));
 		assertEquals(CtAnnotatedElementType.TYPE, annotations.get(0).getAnnotatedElementType());
 
-		List<CtField<?>> fields = enumeration.getValues();
```

Comment:
It is weird you say that because, since the beginning of my contribution to this project, you want to remove all usage of deprecated methods in source and test code.